presto-main/src/test/java/io/prestosql/sql/analyzer/TestAnalyzer.java

```diff
@@ -513,13 +513,6 @@ public void testFetchFirstWithTiesMissingOrderBy()
                 .hasErrorCode(MISSING_ORDER_BY);
     }
 
-    @Test
-    public void testLimitInvalidRowCount()
-    {
-        assertFails("SELECT * FROM t1 LIMIT 987654321098765432109876543210")
```

Comment:
What happened to this case?

Reply:
It doesn't throw `PrestoException` any more.
The row count used to be fetched as text and analysed in the StatementAnalyzer. After this change, a LongLiteral is instantiated in AstBuilder, and `ParsingException` is thrown.
Similarly, negative values now fail in Parser.